includes/ucf-social-shortcode.php

```diff
@@ -0,0 +1,22 @@
+<?php
+/**
+ * Registers the social shortcode
+ * @author RJ Bruneel
+ * @since 1.0
+ * @param array $atts | Assoc. array of shortcode options
+ * @return array
+ **/
+
+if ( ! class_exists( 'UCF_Social_Shortcode' ) ) {
+	class UCF_Social_Shortcode {
+		public static function shortcode( $atts ) {
+			$atts = shortcode_atts( array(
```

Comment:
The shortcode name ('ucf-social-icons') needs to be passed as the third arg of shortcode_atts() for the attribute formatting filters to hook in properly.  https://codex.wordpress.org/Function_Reference/shortcode_atts